packages/mgt-components/src/components/mgt-person-card/mgt-person-card.theme.scss

```diff
@@ -19,7 +19,7 @@
 @import '../../components/mgt-messages/mgt-messages.theme';
 @import '../../components/mgt-file-list/mgt-file-list.theme';
 
-$person-card-background-color: var(--person-card-background-color, var(--neutral-layer-floating));
```

Comment:
this is supposed to be a card and have a background color.

We should probably be looking at using a fluent-card at the top level of the component.